pyamplipi/amplipi.py

```diff
@@ -187,6 +187,37 @@ async def load_preset(self, preset_id: int) -> Status:
     async def announce(self, announcement: Announcement) -> Status:
         response = await self._client.post('announce', announcement.json(**json_ser_kwargs))
         return Status.parse_obj(response)
+    
+    def get_version(self, version_raw: str):
+        version_nums = []
+        tmp = 0
+        for ch in version_raw:
+            if ch.isdigit():
+                tmp = (tmp*10) + int(ch)
+            else:
+                version_nums.append(tmp)
+                tmp = 0
+        version_nums.append(tmp)
+
+        major = version_nums[0]
+        minor = version_nums[1]
+        revision = version_nums[2]
+        return major, minor, revision
+
+    # -- play media call
+    async def play_media(self, media: PlayMedia) -> Status:
+        info = await self.get_info()
+        version_raw = info.version
+        major, minor, revision = self.get_version(version_raw)
+
+        if major <= 0 and (minor < 4 or (minor == 4 and revision < 1)):  # OLD, USE ANNOUNCEMENT
```

Comment:
```suggestion
        version_raw = info.version
        rev = self.get_version(version_raw)

        if rev < MIN_PLAY_MEDIA_VERSION:  # TOO OLD, USE ANNOUNCEMENT
```

Reply:
you'll have to define a MIN_PLAY_MEDIA_VERSION. Uses python tuples for comparison.